curvefs/src/metaserver/partition.cpp

```diff
@@ -295,7 +321,12 @@ MetaStatusCode Partition::InsertInode(const Inode& inode) {
         return MetaStatusCode::PARTITION_ID_MISSMATCH;
     }
 
-    return inodeManager_->InsertInode(inode);
+    const MetaStatusCode& ret = inodeManager_->InsertInode(inode);
+    if (MetaStatusCode::OK == ret) {
+        partitionInfo_.set_inodenum(partitionInfo_.inodenum() + 1);
```

Comment:
There are too many same logics for handle `partitionInfo_` after `CreateInode` or `DeleteInode` success, it's better to wrap them into one function.

Reply:
> There are too many same logics for handle `partitionInfo_` after `CreateInode` or `DeleteInode` success, it's better to wrap them into one function.

fix